src/main/java/com/dft/prestashop/model/shop/Shop.java

```diff
@@ -0,0 +1,22 @@
+package com.dft.prestashop.model.shop;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+@JacksonXmlRootElement(localName = "shop")
```

Comment:
@JacksonXmlRootElement(localName = "shop") should be after @Data

Reply:
moved @JacksonXmlRootElement(localName = "shop") to below @Data annotation